### PR TITLE
Added Moving off Divio article

### PR DESCRIPTION
**Pull request by mebverse**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -54,6 +54,7 @@ General
     :maxdepth: 1
 
     infrastructure
+    move-off-divio
```
Comment (evildmp) [inline index.rst]: Please change to `leaving-divio`

File: move-off-divio.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Moving off Divio
```
Comment (evildmp) [inline move-off-divio.rst]: Title should be `Leaving Divio`

**Review by evildmp — CHANGES_REQUESTED** [review]

File: index.rst (hunk 1 above)
Comment (mebverse) [reply index.rst]: Done

File: move-off-divio.rst (hunk 2 above)
Comment (mebverse) [reply move-off-divio.rst]: Done

**Pull request merged**